lib/matplotlib/axes/_base.py

```diff
@@ -1,4 +1,4 @@
-from collections import OrderedDict
+
```

Comment:
Why did you remove the import for OrderedDict?  Thats why CI is failing.

Reply:
oh damn! Sorry about this! Restoring it back!

Reply:
Totally missed this change in diff!